gallery/src/styles/NxFormLayout/NxFormInlineLayoutExample.tsx

```diff
@@ -4,43 +4,56 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React, { FormEvent } from 'react';
+import React, { useState } from 'react';
 import {faPlus, faMinus} from '@fortawesome/free-solid-svg-icons';
 
-import { NxButton, NxFormGroup } from '@sonatype/react-shared-components';
-import { NxStatefulTextInput } from '@sonatype/react-shared-components';
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { NxFontAwesomeIcon, NxButton, NxButtonBar, NxFormGroup, NxFormRow, nxTextInputStateHelpers, NxStatefulForm, NxTextInput, hasValidationErrors, combineValidationErrors }
+  from '@sonatype/react-shared-components';
+import { } from '@sonatype/react-shared-components';
```

Comment:
out of curiosity, what is the purpose of empty curly brackets with an import?

Reply:
> out of curiosity, what is the purpose of empty curly brackets with an import?

Nothing, I'll remove that.